File: examples/plugin-archives/demo-result-status-table/python/provider.py
def execute(params, upstream, context):
    import pandas as pd
    rows = max(2, int(params.get("rows", 6)))
    scale = float(params.get("scale", 2))
    return pd.DataFrame({"index": list(range(rows)), "value": [index * scale for index in range(rows)]})

File: examples/plugin-archives/demo-result-status-table/js/provider.js
function execute(params, upstream, context, api) {
  const rows = Math.max(2, Math.trunc(Number(params.rows ?? 6)));
  const scale = Number(params.scale ?? 2);
  return api.Table.fromRecords(Array.from({ length: rows }, (_, index) => ({ index, value: index * scale })));
}
